database_admin/migrations/025_tags.up.sql

```diff
@@ -0,0 +1,35 @@
+
+/*CREATE TABLE tag
+(
+    id    INT GENERATED BY DEFAULT AS IDENTITY,
+    key   TEXT NOT NULL CHECK ( NOT EMPTY(formatted_key)),
+    value TEXT NOT NULL,
+    PRIMARY KEY (id, key, value)
+);
+
+CREATE INDEX tags_idx on tag (key, value);
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON tag to listener;
+GRANT SELECT ON tag to evaluator;
+GRANT SELECT ON tag to vmaas_sync;
+GRANT SELECT ON tag to manager;
+
+GRANT SELECT, USAGE ON SEQUENCE public.tag_id_seq TO evaluator;
+GRANT SELECT, USAGE ON SEQUENCE public.tag_id_seq TO listener;
+GRANT SELECT, USAGE ON SEQUENCE public.tag_id_seq TO vmaas_sync;
+*/
+
+CREATE TABLE system_tags
+(
+    tag       TEXT NOT NULL CHECK ( NOT EMPTY(tag)),
```

Comment:
Why not to add tags table?

Reply:
I've tried but I'm hesitant to do it this way because tags are pretty short, and evaluation + querying is much simpler this way. If we ever need to move to a system where we have tags in separate table, it'll be a simple migration.